contrib/building.md

```diff
@@ -266,13 +266,9 @@ The generated docs can be found in `docs/output/realm/<version>/index.html`.
 We've decided to slowly migrate this repository to a mono-repository containing multiple packages (stored in the `./packages` directory). To install and link these, run (from the `realm-js` repo root directory):
 
 ```sh
-npx lerna bootstrap
+npm install --workspaces
 ```
 
-Note: you must successfuly build Realm JS for [iOS](#building-for-ios) and [Android](#building-for-android) before running `lerna`, or the command may fail.
-
-Please familiarise yourself with [Lerna](https://github.com/lerna/lerna) to learn how to add dependencies to these packages.
```

Comment:
Any such doc for `workspaces` that we should reference?

Reply:
I added a link to the NPM docs: https://docs.npmjs.com/cli/v8/using-npm/workspaces